nni/algorithms/compression/v2/pytorch/pruning/basic_pruner.py

```diff
@@ -635,7 +653,7 @@ def trainer(model: Module, optimizer: Optimizer, criterion: Callable[[Tensor, Te
     training_batches : int
         The batch number used to collect activations.
     mode : str
-        'normal', 'dependency_aware' or 'global'.
+        'normal', 'dependency_aware', 'global', 'attention'.
```

Comment:
is 'attention' qualified to be one mode?

Reply:
I think it is a pruning mode because it will prune multiple layers as a whole